### PR TITLE
Update Install stubs + instructions.

### PR DESCRIPTION
**Pull request by JaxkDev**
Small changes to get the stubs inline with the code at the moment they are very outdated (specifically the MySQL scheme).

Feel free to add :)

Updated the following:
- secret-stub (adding the new values)
- mysql (Updating the tables structure and defaults)
- INSTALL.md (Added composer install step)

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 
-0. Before installing, please read [_Can I host Poggit myself?_](README.md#can-i-host-it-myself).
-1. Clone this repository, or download a zipball/tarball from [GitHub](https://github.com/poggit/poggit):
-
-```bash
-git clone https://github.com/poggit/poggit.git
-```
+**Before installing, please read [_Can I host Poggit myself?_](README.md#can-i-host-it-myself).**
 
+0. Clone this repository, or download a zipball/tarball from [GitHub](https://github.com/poggit/poggit):
```
Comment (SOF3) [inline INSTALL.md]: This should start with number 1

File: doc/mysql.sql [hunk 2]
```diff
@@ -7,7 +7,7 @@ CREATE TABLE users (
     email     VARCHAR(511)   DEFAULT '',
     lastLogin TIMESTAMP      DEFAULT CURRENT_TIMESTAMP,
     lastNotif TIMESTAMP      DEFAULT CURRENT_TIMESTAMP,
-    opts      VARCHAR(16383) DEFAULT '{}'
+    opts      VARCHAR(15000) DEFAULT '{}'
```
Comment (SOF3) [inline doc/mysql.sql:10]: What's up here?

**Review by SOF3 — CHANGES_REQUESTED** [review]

File: doc/mysql.sql (hunk 2 above)
Comment (JaxkDev) [reply doc/mysql.sql:10]: Wouldn't allow more then a certain amount all together for the table, forgot the number

File: INSTALL.md (hunk 1 above)
Comment (JaxkDev) [reply INSTALL.md]: 👌

File: doc/mysql.sql (hunk 2 above)
Comment (JaxkDev) [reply doc/mysql.sql:10]: 'Error Code: 1118. Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs'

**Pull request merged**